caikit_nlp/toolkit/text_generation/tgis_utils.py

```diff
@@ -205,7 +202,7 @@ def get_params(
         token_ranks=True,
     )
     stopping = generation_pb2.StoppingCriteria(
-        stop_sequences=stop_sequences or [eos_token] if eos_token else None,
+        stop_sequences=stop_sequences,
```

Comment:
if `stop_sequences` are empty / not provided by the user, does this still work with TGIS?